Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -476,7 +477,60 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newpassword:
     )
 
 
-def main():  # pragma: no cover
+def user_fetch_command(client, secretids):
+    credentials = fetch_credentials(client, secretids)
+    markdown = tableToMarkdown('Credentials to fetch', credentials)
+
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="Delinea.User.Fetch.Credentials",
+        outputs_key_field="credentials",
+        raw_response=credentials,
+        outputs=credentials
+    )

```

Comment:
Why showing the credentials in context? It is unsafe.

Reply:
Removed the code.